poulpe-service/src/main/java/org/jtalks/poulpe/service/transactional/TransactionalUserService.java

```diff
@@ -298,6 +300,17 @@ private PoulpeUser getPoulpeUser(String username) throws NotFoundException {
     @Override
     public void registration(PoulpeUser user) throws ValidationException {
         List<String> errors = new ArrayList<String>();
+
+        BindingResult validateErrors = new BeanPropertyBindingResult(user, "user");
```

Comment:
if I understand correctly, if I understand correctly, it is sufficient use @Valid annotation and delete extra codes?

```
     @Override
     public void registration(@Valid PoulpeUser user) throws ValidationException {
```

I checked, it works, but still want to clarify